### PR TITLE
Valentin

### PR DESCRIPTION
**Pull request by vacrozet**
SDL parse les images en BGR, je les converties en RGB
**Seulement les JPG** .

### CODE REVIEW

File: srcs/sdl/utils.c [hunk 1]
```diff
@@ -54,3 +54,19 @@ Uint32 getpxl(t_param *param, int y, int x)
...
+
+	if (w > 1 || w < 0 || h > 1 || h < 0)
+		return (0);
+	color = (int)*(Uint32 *)(param->texture->pixels + (int)round(param->texture->w * w) * param->texture->pitch + (int)round(param->texture->h * h) * 3);
+	return rgb_color((color) & 0xFF, (color >> 8) & 0xFF, (color >> 16) & 0xFF);
+}
+void	jpg_loader(t_param *param, char *name)
+{
+	IMG_Init(IMG_INIT_JPG);
+	IMG_Quit();
```
[the remark this answers is not in the document]
Comment (vacrozet) [reply srcs/sdl/utils.c]: No
Read this Doc : http://gigi.nullneuron.net/gigilabs/loading-images-in-sdl2-with-sdl_image/

[the remark this answers is not in the document]
Comment (Its-Alex) [reply srcs/sdl/utils.c]: "and before the call to SDL_Quit(), we shut down SDL_Image using IMG_Quit():"
Yes it must be call after :)

**Comment by Its-Alex** [comment]
Now that I think about it I already have implemented `SDL2_Image` https://github.com/ScandalousMan/RT/blob/master/srcs/sdl/init.c#L30 so your function `jpg_loader` is not really usefull

**Comment by Its-Alex** [comment]
I think that adding `param->texture` is not the better way to do this, in my point of view we must load image dynamically and make an array with a dynamic size who contain it. (If we want to load 10 textures at each time)

**Comment by Its-Alex** [comment]
@vacrozet can you rebase before merge ?

**Pull request merged**